cmds/core/dd/dd.go

```diff
@@ -98,19 +102,6 @@ type chunkedBuffer struct {
 }
 
 func init() {
```

Comment:
Do we need empty init function?

Reply:
we don't, remove. Note that the go compiler has always done elision of stuff like this (from the Plan 9 toolchain) but who needs it?